tests/newOffer/newOffer.spec.ts

```diff
@@ -0,0 +1,143 @@
+import { HomePage } from '../../POMs/homePage';
+import { OfferModal } from '../../POMs/offerModal';
+import { NewOfferEnvironments } from '../../environments/newOfferEnvironments';
+
+
+test('no-project',async ({page}) => {
+    const loginPage = new LoginPage(page);
+    const loginEnvironments = new LoginEnvironments(page);
+
+    await page.goto(loginEnvironments.baseUrl);
+    await loginPage.login(loginEnvironments.validEmail, loginEnvironments.validPassword);
+    await loginPage.succesfulLogin();
+    
+    const homePage = new HomePage (page);
+    await homePage.newOfferButton.click();
+
+    const offerModal = new OfferModal (page);
+    const newOfferEnvironments = new NewOfferEnvironments(page); 
+    await offerModal.chooseClient(newOfferEnvironments.clientId0);
+    await offerModal.clientHasNoOffers();
+
+    
+
+});
+
+test('one-project-disabled-project-field', async ({page}) =>{
+    const loginPage = new LoginPage(page);
+    const loginEnvironments = new LoginEnvironments(page);
+
+    await page.goto(loginEnvironments.baseUrl);
+    await loginPage.login(loginEnvironments.validEmail, loginEnvironments.validPassword);
+    await loginPage.succesfulLogin();
+    
+    const homePage = new HomePage (page);
+    await homePage.newOfferButton.click();
+    
+    const offerModal = new OfferModal (page);
+    const newOfferEnvironments = new NewOfferEnvironments(page); 
+    await offerModal.chooseClient(newOfferEnvironments.clientId1);
+    await offerModal.disabledProjectField(); 
+
+});
+
+test('one project disabled continue button until language is choosen', async ({page}) =>{
+    const loginPage = new LoginPage(page);
+    const loginEnvironments = new LoginEnvironments(page);
+
+    await page.goto(loginEnvironments.baseUrl);
+    await loginPage.login(loginEnvironments.validEmail, loginEnvironments.validPassword);
+    await loginPage.succesfulLogin();
+    
+    const homePage = new HomePage (page);
+    await homePage.newOfferButton.click();
+    
+    const offerModal = new OfferModal (page);
+    const newOfferEnvironments = new NewOfferEnvironments(page); 
+    await offerModal.chooseClient(newOfferEnvironments.clientId1);
+    await offerModal.disabledContinueButton(); 
+
+});
+
+test('one-project-create-offer', async ({page}) =>{
+    const loginPage = new LoginPage(page);
+    const loginEnvironments = new LoginEnvironments(page);
+
+    await page.goto(loginEnvironments.baseUrl);
+    await loginPage.login(loginEnvironments.validEmail, loginEnvironments.validPassword);
+    await loginPage.succesfulLogin();
+    
+    const homePage = new HomePage (page);
+    await homePage.newOfferButton.click();
+    
+    const offerModal = new OfferModal (page);
+    const newOfferEnvironments = new NewOfferEnvironments(page); 
+    await offerModal.chooseClient(newOfferEnvironments.clientId1);
+    await offerModal.chooseLanguage(newOfferEnvironments.germanLanguage);
+    await offerModal.createNewOffer();
+    
```

Comment:
How are you sure that creating new offer passed? Could you check it somehow so you know that creation was successful?